torch/csrc/jit/export.cpp

```diff
@@ -499,7 +501,7 @@ GraphEncoder::GraphEncoder(
   for (const std::string& domain : domains_) {
     auto* opset = model_proto_.add_opset_import();
     opset->set_domain(domain);
-    opset->set_version(0);
+    opset->set_version(1);
```

Comment:
Is there a way user can specify the opset version when registering the API. If not, then we should add that.

Reply:
No we currently don't support that. I'll look into it.